test/integration/webhook/v1alpha1/workload_test.go

```diff
@@ -70,3 +70,50 @@ var _ = ginkgo.Describe("Workload defaulting webhook", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("Workload validating webhook", func() {
```

Comment:
`Validating Workload webhooks`

Reply:
I think this is fine. It reads: "Describe Workload validating webhook"